packages/estatico-boilerplate/src/assets/js/helpers/app.js

```diff
@@ -75,25 +75,24 @@ class App {
     return window[namespace].modules[moduleName];
   }
 
-  isInitialised($element, moduleName) {
+  isInitialised(element, moduleName) {
     // jQuery 3 does not allow kebab-case in data() when retrieving whole data object https://jquery.com/upgrade-guide/3.0/#breaking-change-data-names-containing-dashes
-    return $element.data(`${moduleName}Instance`);
+    return element.dataset[`${moduleName}Instance`];
   }
 
   isInitEvent(eventType, moduleName) {
     return window[namespace].modules[moduleName].initEvents.indexOf(eventType) !== -1;
   }
 
   initModules(event) {
-    $('[data-init]').each((key, element) => {
-      const $element = $(element);
-      const modules = $element.data('init').split(' ');
+    [].slice.call(document.querySelectorAll('[data-init]')).forEach((element) => {
```

Comment:
NodeLists have .forEach on their prototype. So there is no need for the [].slice.call hack :)
https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

Reply:
Not in IE11 (and auto-polyfilling is not there yet: https://github.com/zloirock/core-js/issues/329)